internal/server/middleware.go

```diff
@@ -175,6 +199,13 @@ func requireAccessKey(db *gorm.DB, req *http.Request) (access.Authenticated, err
 		return u, fmt.Errorf("identity update fail: %w", err)
 	}
 
+	// TODO(orgs): make org ID required
+	if accessKey.OrganizationID != 0 {
```

Comment:
We should error out here if there's no organization on the access key. Better to play it safe. We can re-visit when we add support roles imo.

Reply:
I'll have to double check, but that may also cause a bunch of test failures. Either way, I'll add this to our list in #2903 so we don't forget.